### PR TITLE
getFullLat method, tests, docs added

### PR DESCRIPTION
**Pull request by mridulnagpal**

### CODE REVIEW

**Comment by mridulnagpal** [comment]
@jywarren close #38 Please have a look

File: spec/javascripts/test_spec.js [hunk 1]
```diff
@@ -51,6 +51,16 @@ describe("Basic testing", function() {
     expect(blurredLocation.getPrecision()).toBe(1);
   });
 
+  it("Checks if getFullLat returns the full latitude of the map", function() {
+    blurredLocation.goTo(45.324324234,-53.32423234234,13);
+    expect(blurredLocation.getFullLat()).toBe(45.324324234);
+  });
+
+  it("Checks if getFullLon returns the full latitude of the map", function() {
+    blurredLocation.goTo(45.324324234,-53.32423234234,13);
```
Comment (jywarren) [inline spec/javascripts/test_spec.js]: Maybe set the precision, then expect getFullLat not to equal getLat!

Comment (mridulnagpal) [inline spec/javascripts/test_spec.js]: Done, please have a look

Comment (mridulnagpal) [inline spec/javascripts/test_spec.js]: We don't have a method to set precision yet, do you think we should have one?

Comment (jywarren) [inline spec/javascripts/test_spec.js]: no, it'll be set by zoom, so to set precision, maybe just zoom to a specific level for now. 

Actually we can later add that as a feature -- basically `setZoomByPrecision()` -- good idea. I'll make an issue.

Comment (mridulnagpal) [inline spec/javascripts/test_spec.js]: @jywarren Yeah though so, have set it according to zoom only. This one's ready to merge, is it?

**Comment by jywarren** [comment]
OK, so if you can just add a check where you change zoom level, confirm getPrecision is changed, and also confirm getLat returns a different length number?

You could even add a check that looks at how much `getPrecision()` changed, and see if `getLat()`'s output is exactly that many digits longer -- convert it to a string with `(output + "").length`.

Make sense?

**Comment by mridulnagpal** [comment]
@jywarren This check is already present in the getLat test spec, I added it when we updated the methods to truncate values.

**Comment by jywarren** [comment]
Super!!! :-)

**Pull request merged**